stl/inc/format

```diff
@@ -1496,6 +1496,218 @@ public:
     }
 };
 
+template <class _Ty>
+class _Fmt_buffer {
+private:
+    _Ty* _Ptr_        = nullptr;

```

Comment:
Sorry, but again this should be `_Ptr`, `_Size` and `_Capacity` for the members and `_Size_` for arguments